app/controllers/miq_policy_controller/policies.rb

```diff
@@ -226,7 +226,8 @@ def excluded_event?(event)
 
   def policy_get_all_folders(parent = nil)
     if !parent.nil?
-      @folders = MiqPolicy::UI_FOLDERS.collect(&:name)
+      # TODO: this is dead code, the one caller passes no arguments
```

Comment:
https://github.com/ManageIQ/manageiq/compare/master...cben:dead-code-in-policy_get_all_folders
will open the PR after this gets merged.